src/fs/migration.test.ts

```diff
@@ -4,13 +4,9 @@
  * SPDX-License-Identifier: MIT
  */
 import { isMigration, parseMigrationFromUrl } from "./migration";
+import { testMigrationUrl } from "./migration-test-data";
 
-// The heart project.
-export const testMigrationUrl =
-  // origin needs to match jest's testUrl
-  "http://localhost/#import:#project:XQAAgACRAAAAAAAAAAA9iImmlGSt1R++5LD+ZJ36cRz46B+lhYtNRoWF0nijpaVyZlK7ACfSpeoQpgfk21st4ty06R4PEOM4sSAXBT95G3en+tghrYmE+YJp6EiYgzA9ThKkyShWq2UdvmCzqxoNfYc1wlmTqlNv/Piaz3WoSe3flvr/ItyLl0aolQlEpv4LA8A=";
-
-describe("parseMigrationFromUrl", () => {
+describe.skip("parseMigrationFromUrl", () => {
```

Comment:
Does this test still need skipping?
I removed the skip and ran it locally, and it passed :)

```suggestion
describe("parseMigrationFromUrl", () => {
```

Reply:
Oops, no! This was disabled until I worked around the LZMA issues.